src/flavors/duwamish_flavor/static/data/areas-for-discussion.json

```diff
@@ -0,0 +1 @@
+tureCollection","totalFeatures":1,"features":[{"type":"Feature","id":"Areas-for-discussion.1","geometry":{"type":"MultiPolygon","coordinates":[[[[1270984.9935023915,206776.874548335],[1271876.7958930065,206754.57948856964],[1272657.1229847947,206553.92395068126],[1273259.08959846,206197.20299443527],[1273682.695734002,205528.35120147403],[1273905.6463316556,204547.36857179756],[1273236.7945386944,203989.9920776632],[1272634.8279250292,203789.33653977484],[1271609.2551758222,203945.40195813245],[1270605.9774863801,204034.58219719396],[1269781.0602750613,204212.94267531694],[1269600.5058681848,204695.48841034522],[1269535.8146176422,205639.8265003009],[1269758.765215296,206553.92395068126],[1270984.9935023915,206776.874548335]]]]},"geometry_name":"the_geom","properties":{"Id":0}}],"crs":{"type":"name","properties":{"name":"urn:ogc:def:crs:EPSG::2926"}}}
```

Comment:
Do we need these files? Perhaps they were accidentally added to the index when you committed?